content/id/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Dalam bidang robotika dan otomatisasi, _control loop_ adalah
+lingkaran tertutup yang mengatur keadaan suatu sistem.
+
+Berikut adalah salah satu contoh loop kontrol: termostat di sebuah ruangan.
+
+Ketika Anda mengatur suhunya, itu mengisyaratkan ke termostat
+tentang *keadaan yang Anda inginkan*. Sedangkan suhu kamar yang sebenarnya 
+adalah *keadaan saat ini*. Thermostat berfungsi untuk membawa keadaan saat ini
+mendekati ke keadaan yang diinginkan, dengan menghidupkan atau mematikan 
+perangkat.
+
+Di Kubernetes, controller adalah kontrol tertutup yang mengawasi keadaan cluster
+{{< glossary_tooltip term_id="cluster" >}} Anda, lalu membuat atau meminta 
+perubahan jika diperlukan. Setiap controller mencoba untuk memindahkan status 
+cluster saat ini mendekati keadaan yang diinginkan.
+
+{{< glossary_definition term_id="controller" length="short">}}
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pola controller 
+
+Sebuah controller melacak sekurang-kurangnya satu jenis *resources* dari 
+Kubernetes.
+[objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/) ini
+memiliki *spec field* yang merepresentasikan keadaan yang diinginkan. Satu atau 
+lebih controller untuk *resource* tersebut bertanggung jawab untuk membuat 
+keadaan sekarang mendekati keadaan yang diinginkan.
+
+Controller mungkin saja melakukan tindakan itu sendiri; namun secara umum, di 
+Kubernetes, controller akan mengirim pesan ke
+{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} yang 
+mempunyai efek samping yang bermanfaat. Anda bisa melihat contoh-contoh 
+di bawah ini.
+
+{{< comment >}}
+Beberapa controller bawaan, seperti controller *namespace*, bekerja pada objek
+yang tidak memiliki *spec*. Agar lebih sederhana, halaman ini tidak 
+menjelaskannya secara detail.
+{{< /comment >}}
+
+### Kontrol melalui server API
+
+Controller {{< glossary_tooltip term_id="job" >}} adalah contoh dari controller
+bawaan dari Kubernetes. Controller bawaan tersebut mengelola status melalui
+interaksi dengan server API dari suatu cluster.
+
+Job adalah *resource* dalam Kubernetes yang menjalankan a
+{{< glossary_tooltip term_id="pod" >}}, atau mungkin beberapa Pod sekaligus, 
+untuk melakukan sebuah pekerjaan dan kemudian berhenti.
+
+(Setelah [dijadwalkan](../../../../en/docs/concepts/scheduling/), objek Pod 
```

Comment:
need to refer to ID docs
```suggestion
(Setelah [dijadwalkan](../../../../id/docs/concepts/scheduling/), objek Pod 
```

Reply:
```suggestion
(Setelah [dijadwalkan](/id/docs/concepts/scheduling/), objek Pod
```

Reply:
When I create this PR the folder `/id/docs/concepts/scheduling/` is not there. So, i believe it will become a broken link.